be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -365,9 +369,12 @@ Status HdfsOrcScanner::do_open(RuntimeState* runtime_state) {
             VLOG_FILE << "[ORC] lazy load field = " << it->col_name();
             _lazy_load_ctx.lazy_load_slots.emplace_back(it);
             _lazy_load_ctx.lazy_load_indices.emplace_back(src_slot_index);
+            _lazy_load_ctx.lazy_load_orc_positions.emplace_back(0);
```

Comment:
add some comment for emplace_back(0) to explain why

Reply:
It reserves room for following code to set correct value.